ludwig/features/sequence_feature.py

```diff
@@ -16,7 +16,9 @@
 # ==============================================================================
 import os
 
+import dask.dataframe as dd
```

Comment:
Same thing as above regarding the import guard.

Reply:
Fixed.